frontend/src/app/main/ui/workspace/sidebar/assets/file_library.cljs

```diff
@@ -384,7 +385,15 @@
         (mf/use-fn
          (mf/deps file-id)
          (fn []
-           (st/emit! (dw/unselect-all-assets file-id))))]
+           (st/emit! (dw/unselect-all-assets file-id))))
+
+        count-variants
+        (mf/use-fn
+         (mf/deps library)
+         (fn [variant-id]
+           (->> (ctkl/components-seq library)
+                (filter #(= variant-id (:variant-id %)))
+                count)))]
```

Comment:
use filterv here, because you intention is counting it, and count will iterate again the seq for count, so if you generate a vector, then the count will be for free.